chore(docs) Remove personal Language instances (#463)
pyth-network/documentation: pages/price-feeds/how-pyth-works/price-aggregation.mdx

![](../../../images/Price_Aggregation_1.png)

Second, the **aggregate price should appropriately weight data sources with different levels of accuracy.** Pyth allows publishers to submit a confidence interval because they have varying levels of accuracy in observing the price of a product. This property can result in situations where one publisher reports a price of \$101 +/- 1, and another reports \$110 +/- 10. In these cases, aggregating the price to be closer to \$101 than \$110 is appropriate, as in the figure below.

![](../../../images/Price_Aggregation_2.png)

Finally, the **aggregate confidence interval should reflect the variation between publishers' prices.**
Under normal market conditions, it is reasonable to expect a product to trade at a similar price across exchanges.

> In these cases, aggregate confidence interval will be preferred to reflect the confidence intervals of the individual data providers, as shown in the figure on the left.

Reviewer: ```suggestion
In these cases, aggregate confidence interval is preferable to reflect the confidence intervals of the individual data providers, as shown in the figure on the left.
```

This process acts like a hybrid between a mean and a median, giving confident publishers more influence, while still capping the maximum influence of any single publisher.
The algorithm has an interpretation as computing the minimum of an objective function that penalizes the aggregate price from deviating too far from the publishers' prices.

> This interpretation allows proving properties of the algorithm's behavior: for example, the aggregate price will always lie between the 25th and 75th percentiles of the publishers' prices.

Reviewer: ```suggestion
This interpretation allows properties of the algorithm's behavior to be proved: for example, the aggregate price will always lie between the 25th and 75th percentiles of the publishers' prices.
```
